src/tests/kas.test.ts

```diff
@@ -0,0 +1,23 @@
+import { parse, compare } from '@khanacademy/kas'
+import { ComputeEngine } from '@cortex-js/compute-engine'
+
+test('Test kas with compute engine and edge cases', async () => {
+  const options = {
+    simplify: false,
+    form: true
+  }
+  const ce = new ComputeEngine()
+
+  const consideredEqual = [
+    { expr1: ce.serialize(ce.parse('\\frac42')), expr2: ce.serialize(ce.parse('\\frac{4}{2}')) },
+    { expr1: 'x>4', expr2: '4<x' },
+    { expr1: 'x\\le4', expr2: '4\\gex' },
+    { expr1: '|-4|', expr2: '|4|' },
+    { expr1: '\\sqrt[3]{27}', expr2: '\\sqrt[3]{27}' },
```

Comment:
Does this test case mean we can support and should add the nth root to the keyboard?

Reply:
Yeah, you can go ahead and do that by updating the layout definition [here](https://github.com/openstax/k12-apps-raise/blob/592125cf97a1f81253061f1ff4603f12ce03d927/src/components/Mathfield.tsx#L6).

Reply:
When we were originally trying to add the nth root key to the keyboard it was displaying oddly on the keyboard and in the input field. I couldn't figure out how the mathlive demo keyboard was displaying the nth root key properly, but that might be a place to look again if you run into the same issue.

Reply:
Just a thought, but if it has to do with fonts, it could be because we do [this](https://github.com/openstax/k12-apps-raise/blob/592125cf97a1f81253061f1ff4603f12ce03d927/src/components/Mathfield.tsx#L4) while their demos probably use their fonts.

Reply:
@P-Gill97 - Can you try setting that variable to `https://unpkg.com/mathlive/dist/fonts/` to see if it helps. I noticed [here](https://cortexjs.io/mathlive/guides/customizing/#fonts) it specifically says you need to user their fonts :joy: . If it works, we should confirm whether that URL is okay to use (it looks like it's a CDN :man_shrugging: ).

Reply:
Just confirming that it's a reliable URL that's okay for "production". I'm not familiar with `unpkg.com`, but from a cursory look it seems like it would be okay. Just wanted another pair of 👀 to confirm. Otherwise we can figure out to host it somewhere as well 🤷‍♂️ .